types/backend/index.d.ts

```diff
@@ -1 +1,4 @@
-/// <reference path="../common/declaration.d.ts" />
\ No newline at end of file
+/// <reference path="../common/declaration.d.ts" />
+declare namespace $w {
```

Comment:
I remember we saw that adding the namespace on the backend had some undesired side-effects (such as suggesting $w.$w function). Can we improve it? Can you remind me why we need this definition?